diskv.go

```diff
@@ -40,6 +42,11 @@ var (
 	errImportDirectory       = errors.New("can't import a directory")
 )
 
+func init() {
+	// Make sure we have good random numbers
+	rand.Seed(time.Now().UnixNano())
+}
```

Comment:
-1 on any `func init` — take a Seed as an option, default to `time.Now().UnixNano()` if you like, and build a rand.New based on that seed in the Diskv.